pkg/gds/emails/mock.go

```diff
@@ -139,7 +139,7 @@ func (c *mockSendGridClient) Send(msg *sgmail.SGMailV3) (rep *rest.Response, err
 		}, err
 	}
 
-	// Email recepients are stored in Personalizations
+	// Email recipients are stored in Personalizations
```

Comment:
Thanks for correcting my spelling mistakes!

Reply:
Code spell checker is super helpful! https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker